Src/Support/Google.Apis.Auth/OAuth2/JsonCredentialParameters.cs

```diff
@@ -53,6 +53,13 @@ public class JsonCredentialParameters
         [Newtonsoft.Json.JsonProperty("project_id")]
         public string ProjectId { get; set; }
 
+        /// <summary>
+        /// Project ID associated with this credential for the purposes
+        /// of quota calculations and billing.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty("quota_project")]
+        public string QuotaProject { get; set; }
```

Comment:
Is anything actually using this? I'd expect to see it being read in a call to `new UserCredential(...)` but I can't see that right now.

Reply:
Aha - sorry for missing that.